README.md

```diff
@@ -100,19 +100,19 @@ Find a demo in the Examples folder.
 	```
 	The scroll direction of the grid. The grid layout scrolls along one axis only, either horizontally or vertically. The default value of this property is `vertical`.
 	
-- **isFistCellExcluded**:
+- **isFirstCellExcluded**:
```

Comment:
Thx @kandelvijaya !

Well i should fixe the typo for the property name 😞

Reply:
i created the associated issue #5. I you've time you could update your PR. Otherwise i ll do it. What do u prefer ? 
Thx again

Reply:
I made the changes in the code. 👍